CLI Panics when malformed attribute is input

<!--
Thanks for submitting a bug report. Please fill the template below,
otherwise we will not be able to process this bug report.
-->

#### Summary
<!-- Summarize the problem in a few sentences: -->

CLI Panics when a malformed attribute is used on an `update`.

#### Steps to Reproduce
<!-- How can we reproduce the problem? -->

1.  Run `ttn-lw-cli gateways set eui-58a0cbfffe8008a0 --attributes a`


<!--
Please upload relevant configuration (as .txt).
If you use the command "ttn-lw-stack config", you can redact sensitive config.
-->

#### What do you see now?
<!--
Please paste terminal output, upload logs (as .txt) or upload screenshots.
-->

```
panic: runtime error: index out of range

goroutine 1 [running]:
go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/commands.mergeAttributes(0x0, 0xc0001a1100, 0xc0004d4a00)
    /home/travis/gopath/src/go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/commands/flags.go:74 +0x2f8
go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/commands.glob..func56(0x2b02ec0, 0xc000142060, 0x1, 0x3, 0x0, 0x0)
    /home/travis/gopath/src/go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/commands/gateways.go:263 +0x22f
github.com/spf13/cobra.(*Command).execute(0x2b02ec0, 0xc000297f80, 0x3, 0x3, 0x2b02ec0, 0xc000297f80)
    /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0x2b07ac0, 0x0, 0x0, 0x1c57b6c)
    /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
    /home/travis/gopath/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
main.main()
    /home/travis/gopath/src/go.thethings.network/lorawan-stack/cmd/ttn-lw-cli/main.go:27 +0x32
```
#### What do you want to see instead?
<!-- Please add some examples or mock-ups if applicable. -->

An error since the input in invalid but don't panic (pun intended).

#### Environment
<!--
Your environment: OS/Browser/Gateway/Device/...? Versions? IDs/EUIs?
Paste the output of "ttn-lw-cli version" or "ttn-lw-stack version" if applicable.
-->

```
The Things Network Command-line Interface: ttn-lw-cli
Version:             3.0.3
Go version:          go1.12.5
OS/Arch:             darwin/amd64
```

#### How do you propose to implement this?
<!-- Please think about how this could be fixed. -->

Need to look into how `cobra` handles this.

#### Can you do this yourself and submit a Pull Request?
<!-- You can also @mention experts if you need help with this. -->

Yup

--- a/cmd/ttn-lw-cli/commands/flags.go
+++ b/cmd/ttn-lw-cli/commands/flags.go
@@ -71,6 +71,9 @@ func mergeAttributes(attributes map[string]string, flagSet *pflag.FlagSet) map[s
 	}
 	for _, kv := range kv {
 		kv := strings.SplitN(kv, "=", 2)
+		if len(kv) != 2 {
+			continue
+		}
 		if kv[1] == "" {
 			delete(out, kv[0])
 		} else {

